helios-integrations/helios-retrofit/src/main/kotlin/helios/retrofit/JsonableEvidence.kt

```diff
@@ -0,0 +1,6 @@
+package helios.retrofit
+
+import helios.typeclasses.Decoder
+import helios.typeclasses.Encoder
+
+typealias JsonableEvidence<T> = Triple<Class<T>, Encoder<T>, Decoder<T>>
```

Comment:
Where does Triple come from? Can you use Tuple3 from Arrow instead?

Reply:
Triple is from stdlib. Can Tuple3 be destructured?

Reply:
Yep!